tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -17,6 +17,48 @@ jobs:
       displayName: CPU EP, Build and Test on Android Emulator
     - script: /bin/bash tools/ci_build/github/android/run_nnapi_code_coverage.sh $(pwd)
       displayName: NNAPI EP, Build, Test and Get Code Coverage on Android Emulator
+    - task: PublishPipelineArtifact@0
+      displayName: 'Publish code coverage report'
+      inputs:
+        artifactName: "coverage_rpt.txt"
+        targetPath: '$(Build.SourcesDirectory)/build_nnapi/Debug/coverage_rpt.txt'
+        publishLocation: 'pipeline'
     - script: /bin/bash tools/ci_build/github/linux/ort_minimal/nnapi_minimal_build_minimal_ort_and_run_tests.sh $(pwd)
       # Build Minimal ORT with NNAPI and reduced Ops, run unit tests on Android Emulator
       displayName: Build Minimal ORT with NNAPI and run tests
+
+- job: Update_Dashboard
```

Comment:
Do we want to report code coverage only for merge commit to avoid noises?
See #5913

Reply:
Image is size is different from the code coverage as just one line changes could in theory can change the control-flow significantly. We are not only interested whether that one line is exercised or not. It is also harder to limit the code coverage just the code change. Automating it is not straightforward.

Reply:
The problem is if you report a number for every commit, this will create tons of noise, which make pin-point the PR causing problem very hard. Usually we do not expect devs to check the code coverage percentage after every commit, just as we don't expect devs to check if their changes causing binary size increase, but we do want to know which PR caused it afterward, for this, having a record for each commit will make this very hard.